File: JavaScript/453. Minimum Moves to Equal Array Elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minMoves = function(nums) {
    var res = 0;
	var m = Number.MAX_VALUE;
	for(var i = 0; i<nums.length; i++){
        m = Math.min(m,nums[i]);
    }
	for(var i = 0; i<nums.length; i++){
        res+=nums[i]-m;
    }
	return res;
};

File: C/453. Minimum Moves to Equal Array Elements.c
int minMoves(int* nums, int numsSize){
    int res = 0;
	int m = INT_MAX;
	for(int i = 0; i<numsSize; i++){
        m = fmin(m,nums[i]);
    }
	for(int i = 0; i<numsSize; i++){
        res+=nums[i]-m;
    }
	return res;
}
